crates/core/src/db/datastore/traits.rs

```diff
@@ -30,6 +30,8 @@ pub struct TxRecord {
     pub(crate) key: DataKey,
     /// The table that was modified.
     pub(crate) table_id: TableId,
+    /// The table that was modified.
+    pub(crate) table_name: String,
```

Comment:
Note the addition of an owned string to `TxRecord`. Table metrics such as the number of inserts and deletes are recorded on commit. Not exactly ideal, but good enough for now I suppose.